test/e2e/csv_e2e_test.go

```diff
@@ -4169,6 +4175,210 @@ var _ = Describe("ClusterServiceVersion", func() {
 
 		Expect(apiService2.Spec.CABundle).Should(Equal(apiService1.Spec.CABundle))
 	})
+
+	It("Disabling copied csvs", func() {
+		ns := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: genName("csv-toggle-test-"),
+			},
+		}
+
+		// Create a new operator group for the new namespace
+		operatorGroup := v1.OperatorGroup{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      genName("csv-toggle-test-"),
+				Namespace: ns.GetName(),
+			},
+		}
```

Comment:
We could also use the `SetupGeneratedTestNamespace` function to create a testing namespace and an OperatorGroup resource that targets that namespace:

```go
var (
		generatedNamespace corev1.Namespace
)
BeforeEach(func() {
		generatedNamespace = SetupGeneratedTestNamespace(genName("csv-toggle-test-"))
}
AfterEach(func() {
		TeardownNamespace(generatedNamespace.GetName())
}
```

Reply:
Yep, we'd just need to update that method to allow users to provide the set of targetNamespaces.